comment/views.py

```diff
@@ -0,0 +1,84 @@
+from django.shortcuts import render
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from post.models import Post
+from post.serializers import PostSerializer
+from .models import Comment
+from .serializers import CommentSerializer
+
+
+class CommentListView(APIView):
+  
+  
+  def get(self, request):
+    try:
+      post_id=request.data.get('post')
+    except:
+      return Response({"detail": "missing fields['post]"}, status=status.HTTP_400_BAD_REQUEST)
+    
+    try:
+      commentlist=Comment.objects.filter(post_id=post_id)
```

Comment:
```suggestion
      commentlist=Comment.objects.filter(post_id=post_id).exists()
```
filter는 조건에 맞는 데이터가 없을 경우 에러가 아닌, 빈 배열을 반환합니다 ! 즉 try에서 걸리지 않고 통과해요 !
(참고 : https://velog.io/@mquat/django-get%EA%B3%BC-filter)